src/scan.l

```diff
@@ -285,9 +285,24 @@ M4QEND      "]""]"
 }
 
 <CODEBLOCK_MATCH_BRACE>{
+    "'}'"      |
```

Comment:
Can you simplify `"'{'" | "'}'"` to this?
```
'[{}]'
```

Reply:
Yes - what does this mean in terms of table size and performance?